lib/shopify_api/resources/fulfillment_order.rb

```
@@ -22,6 +22,11 @@ def self.find(scope, *args)
       end
     end
 
+    def self.release_hold(order_id:)
```

I wonder this should raise an exception if called for an unsupported version of the API?

---

good point, I am going to add a minimum supported version 👍